### DISCUSSION TITLE
Warning about search.jpope.org instance

### DISCUSSION THREAD

**Discussion by unixfox** [searxng/searx-instances#84] [General]
Hello @jpope777

Please adapt your anti bot configuration so that our uptime tracker can check the availability of your instance: https://github.com/searxng/searx-instances-uptime

You should probably just whitelist the path `/` and that's it. If you fail to allow the uptime tracker your instance will soon have a displayed uptime of 0% on the official searx instances list (searx.space).

**Comment by jpope777** [comment]
Thanks @unixfox, However, I'm not changing my anti bot configurations. Have far too many bots crawling my site as it is. If my instance needs to be and is removed from the public lists, that's completely fine with me.
Sorry if it has caused you any inconvenience.
